src/server/auth-client.ts

```diff
@@ -404,7 +444,24 @@ export class AuthClient {
 
     // Set response and save transaction
     const res = NextResponse.redirect(authorizationUrl.toString());
-    await this.transactionStore.save(res.cookies, transactionState);
+
+    // Apply transaction cookie management based on configuration
+    if (reqCookies) {
+      if (!this.enableParallelTransactions) {
+        // When parallel transactions are disabled, delete all existing transaction cookies
+        await this.transactionStore.deleteAll(reqCookies, res.cookies);
```

Comment:
Can we, instead of doing this, ensure the cookie name is not unique but we use a single transaction cookie name [here](https://github.com/auth0/nextjs-auth0/blob/d916c1823b69730c47971ed7a9026412a222a7be/src/server/transaction-store.ts#L75)?